love/src/components/ScriptQueue/ConfigPanel/ConfigPanel.module.css

```diff
@@ -446,3 +446,63 @@ fieldset p {
 .element {
   margin: 0.25rem;
 }
+
+:global(.control-label) {
+  padding-top: 7px;
+  margin-bottom: 0;
+  text-align: right;
+}
+:global(.form-group) {
+  margin-bottom: 15px;
+}
+:global(.form-control) {
+  width: 95%;
+  box-sizing: border-box;
+  border-radius: 3px;
+  padding: 0.4em;
+  font-family: inherit;
+  font-size: inherit;
+  background-color: transparent;
+  border: 1px solid var(--secondary-font-dimmed-color);
+  color: inherit;
+  color: var(--base-font-color);
+}
+:global(.form-control:focus) {
+  outline: none;
+  border: 1px solid var(--secondary-font-color);
+}
+:global(.row) {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+}
+:global(.col-xs-9) {
+  width: calc(100% / 12 * 9 );
+}
+:global(.col-xs-5) {
+  width: calc(100% / 12 * 5 );
+}
+:global(.col-xs-3) {
+  width: calc(100% / 12 * 3 );
+}
+:global(.col-xs-2) {
+  width: calc(100% / 12 * 2 );
+}
+:global(.array-item-list) {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: row;
+  flex-direction: row;
+  padding-left: 0;
+  margin-bottom: 0;
+}
+:global(.array-item) {
+  position: relative;
+  display: block;
+  padding: 0.5rem 1rem;
+  text-decoration: none;
+  border: 1px solid rgba(0,0,0,.125);
+}
```

Comment:
Thanks for refactoring this! I have one last suggestion though:
As you are setting global styles, I recommend to prepend the class of the parent which contains all these elements. One example:
```
.sidePanel :global(.control-label) {
  padding-top: 7px;
  margin-bottom: 0;
  text-align: right;
}
``` 
This way you avoid that other components with same global classes get affected if both of them are present on the same view (the `.sidePanel` is converted to an unique identifier of the component by the CSS modules).